PR: https://github.com/odoo/odoo/pull/80434

From: 6a23b9bcd6aea529eb339640e6231e1aa4c9e207
From: William Henrotin
Date: 2021-12-07 16:20:05

Structural Changes: 4
Total Changes: 84

[REF] *stock*: rename model `stock.production.lot` into `stock.lot`

Task: 2648449
Part-of: odoo/odoo#80434

================================= pseudo patch: =================================

--- a/addons/repair/models/__init__.py
+++ b/addons/repair/models/__init__.py
@@ -3,5 +3,5 @@
 
 from . import repair
 from . import stock_traceability
-from . import stock_production_lot
+from . import stock_lot
 from . import account_move

--- a/addons/repair/models/repair.py
+++ b/addons/repair/models/repair.py
@@ -72,7 +72,7 @@ class Repair(models.Model):
         help="This is the location where the product to repair is located.",
         states={'draft': [('readonly', False)], 'confirmed': [('readonly', True)]})
     lot_id = fields.Many2one(
-        'stock.production.lot', 'Lot/Serial',
+        'stock.lot', 'Lot/Serial',
         domain="[('product_id','=', product_id), ('company_id', '=', company_id)]", check_company=True,
         help="Products repaired are all belonging to this lot")
     guarantee_limit = fields.Date('Warranty Expiration', states={'confirmed': [('readonly', True)]})
@@ -655,7 +655,7 @@ class RepairLine(models.Model):
         'stock.move', 'Inventory Move',
         copy=False, readonly=True)
     lot_id = fields.Many2one(
-        'stock.production.lot', 'Lot/Serial',
+        'stock.lot', 'Lot/Serial',
         domain="[('product_id','=', product_id), ('company_id', '=', company_id)]", check_company=True)
     state = fields.Selection([
         ('draft', 'Draft'),

--- a/addons/repair/models/stock_production_lot.py
+++ b/addons/repair/models/stock_lot.py
@@ -3,8 +3,8 @@
 from collections import defaultdict
 from odoo import api, fields, models, _
 
-class ProductionLot(models.Model):
-    _inherit = 'stock.production.lot'
+class StockLot(models.Model):
+    _inherit = 'stock.lot'
 
     repair_order_ids = fields.Many2many('repair.order', string="Repair Orders", compute="_compute_repair_order_ids")
     repair_order_count = fields.Integer('Repair order count', compute="_compute_repair_order_ids")
